messages/messages_test.go

```diff
@@ -2,33 +2,158 @@ package messages
+		name string
+		args args
+		want string
+	}{
+		{"with full URL", args{"http://example.com"}, "[Click here](http://example.com)"},
+		{"with short URL", args{"example.com"}, "[Click here](example.com)"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := ClickHere(tt.args.url); got != tt.want {
+				t.Errorf("ClickHere() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestLimitExceeded(t *testing.T) {
+	type args struct {
+		limit    int
+		artworks int
+		count    int
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"with a single artwork", args{10, 1, 5}, "Album size `(5)` is higher than the server's limit `(10)`, album has been cut."},
+		{"with multiple artworks", args{10, 3, 5}, "Album size `(5)` is higher than the server's limit `(10)`, only the first image of every artwork has been sent."},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := LimitExceeded(tt.args.limit, tt.args.artworks, tt.args.count); got != tt.want {
+				t.Errorf("LimitExceeded() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestCrosspostBy(t *testing.T) {
+	type args struct {
+		author string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"with some author name", args{"nabi"}, "Crosspost requested by nabi"},
+		{"with empty author name", args{""}, "Crosspost requested by anonymous"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := CrosspostBy(tt.args.author); got != tt.want {
+				t.Errorf("CrosspostBy() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestRateLimit(t *testing.T) {
+	type args struct {
+		duration time.Duration
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"with limit of 0", args{time.Second * 0}, "Hold your horses! You're getting rate limited. Try again in **0s**"},
+		{"with limit of 1 second", args{time.Second}, "Hold your horses! You're getting rate limited. Try again in **1s**"},
+		{"with limit of 1 minute", args{time.Minute}, "Hold your horses! You're getting rate limited. Try again in **1m0s**"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := RateLimit(tt.args.duration); got != tt.want {
+				t.Errorf("RateLimit() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestListChannels(t *testing.T) {
+	type args struct {
+		channels []string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"with single channel", args{[]string{"theoffice"}}, "<#theoffice> | `theoffice`"},
+		{"with multiple channels", args{[]string{"theoffice", "lounge", "memes"}}, "<#theoffice> | `theoffice` • <#lounge> | `lounge` • <#memes> | `memes`"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := ListChannels(tt.args.channels); got != tt.want {
+				t.Errorf("ListChannels() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
```

Comment:
Even though tests in this package were using testify, as you can see [here](https://github.com/VTGare/boe-tea-go/blob/master/artworks/twitter/twitter_suite_test.go) new tests are written with Ginkgo test framework. Testify is fine, but I'd like to fully migrate to Ginkgo instead to keep things consistent.